pkg/cmd/root.go

```diff
@@ -52,7 +52,7 @@ func NewRootCmd() (*RootCmd, error) {
 		flags: f,
 		cmd: &cobra.Command{
 			Use:          constants.AppName,
-			Short:        "RHTAP Installer CLI",
+			Short:        "Software Supply Chain installer CLI",
```

Comment:
Remove installer or CLI, maybe cli is better long term as the command line can install things today but may orchestrated things already installed later, or help with observability in the future. Basically installer is just one function of the cli

Reply:
Done